tools/scalar-schema/README.md

```diff
@@ -0,0 +1,80 @@
+# Schema Tool for Scalar DB
+This tool makes schemas on Cassandra or Cosmos DB for [Scalar DB](https://github.com/scalar-labs/scalardb).
+  - This tool creates databases(collections) and tables(containers), also inserts metadata which is required by Scalar DB for Cosmos DB
```

Comment:
Can you have a corresponding description for Cassandra?

Reply:
Thank you! I added a description.